src/screens/Home/Home.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react'
+import { Anchor, Box, Grid, Image, Text } from 'grommet'
+import Kelp from 'images/kelp.png'
+import Title from 'images/title.svg'
+import Zooniverse from 'images/zooniverse.png'
+import ResponsiveImage from 'components/ResponsiveImage'
+import styled from 'styled-components'
+import ChooseLocation from './components/ChooseLocation'
+
+const StyledText = styled(Text)`
+  font-family: Neuton;
+`
+
+export default function Home() {
+  return (
+    <Box background='sand' direction='row' fill>
+      <Grid
+        rows={['1/2', '1/2']}
+        columns={['1/4', '3/4']}
+        areas={[
+          { name: 'banner', start: [0, 0], end: [0, 1] },
+          { name: 'intro', start: [1, 0], end: [1, 1] },
+          { name: 'body', start: [1, 1], end: [1, 1] }
+        ]}
+      >
+        <Box gridArea='banner' border={{ color: 'kelp', side: 'right', size: 'large' }}>
+          <Image gridArea='banner' fit='cover' src={Kelp} />
+        </Box>
+
+        <Box
+          gridArea='intro'
+          margin={{ horizontal: 'medium', top: 'auto' }}
+          width={{ max: '40rem' }}
+        >
+          <ResponsiveImage height='1.25rem' src={Zooniverse} />
+          <ResponsiveImage src={Title} />
```

Comment:
Is there a reason why this is an image and not headers? The text is inaccessible right now and text in an image should be avoided for this reason. Why not make these headers and styled accordingly?

Reply:
This is a custom font that was further modified in illustrator to make it a logo. At this point, it doesn't directly mirror a font. I need to add alt text to all the images in this PR. However, if alt text does not suffice here, we'll have to change the logo design.

Reply:
Ok, I'd suggest taking a look at how we did this for the `ZooniverseLogoType` and use that as a model on how to make this accessible. There should be an H1 heading somewhere for the app, however, even if this stays an SVG image.